pydicer/convert/data.py

```diff
@@ -278,14 +311,16 @@ def convert(self, patient=None):
                     rt_struct_file = df_files.iloc[0]
 
                     # Get the linked image
-                    linked_uid = rt_struct_file.referenced_uid
+                    # Need to disable this pylint check here only, seems to be a bug in
+                    # pylint/pandas
+                    # pylint: disable=unsubscriptable-object
```

Comment:
IS this something we should be concerned about? We could investigate it further if so...

Reply:
I don't think we need to worry, I was reading a bit about it here and it seems to be something that has popped up in certain versions of pylint etc: https://github.com/PyCQA/pylint/issues/4577

There might be a cleaner workaround than to disable the check? But this only disables the pylint check for these lines of code...